src/Utils/Logger.ts

```diff
@@ -34,6 +35,17 @@ export class Logger {
       return;
     }
 
+    captureException(message, {
```

Comment:
Seems it is better to use the `captureMessage` instead

Reply:
Yeah, let's use `captureMessage`, but we still need to enable `attachStacktrace`, which leads to using fingerprint, which overall leads to behaviour similar to `captureException`

Reply:
@derevnjuk added sdn rotation logic:

It will try to fetch cli config with 1hour timeout – if it fails within 1.5 secs the next attempt will be in 5 mins.
It will always fallback to "hardcoded" sdn token.
Rotations interval is set to 1hr.

If this is too much for this PR to be reviewed, let's split it, but we will need alpha channel for this.